app/views/idv/doc_auth/agreement.html.erb

```diff
@@ -19,18 +19,23 @@
                        url: url_for,
                        method: 'put',
                        html: { autocomplete: 'off', class: 'margin-top-2 js-consent-continue-form' } do |f| %>
-  <%= check_box_tag(
-        :ial2_consent_given,
-        true,
-        false,
-        class: 'usa-checkbox__input',
-      ) %>
-  <label class="usa-checkbox__label margin-bottom-4" for="ial2_consent_given">
-    <%= t('doc_auth.instructions.consent') %>
-    <%= new_window_link_to t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url %>
-  </label>
+  <div class="margin-bottom-4">
+    <%= check_box_tag(
+          :ial2_consent_given,
+          true,
+          false,
+          class: 'usa-checkbox__input',
+        ) %>
+    <label class="usa-checkbox__label" for="ial2_consent_given">
+      <%= t('doc_auth.instructions.consent') %>
+      <%= new_window_link_to t('doc_auth.instructions.learn_more'), MarketingSite.security_and_privacy_practices_url %>
+    </label>
+    <% if flow_session[:error_message] %>
+      <%= render 'shared/ial2-alert' %>
```

Comment:
The partial is quite small and not very reusable. Could we just render the markup inline here instead?

Reply:
I considered rendering inline, but the [rails documentation](https://guides.rubyonrails.org/layouts_and_rendering.html#using-render-with-inline) deems this to be a bad practice. Also, it makes the code hard to maintain for future us.

Reply:
Sorry, I was unclear. By "inline", I just mean having the markup here, i.e.

```suggestion
      <div class="usa-error-message usa-error-message--with-icon" role="alert">
        <%= t('sign_up.agree_to_rules_of_use_and_continue') %>
      </div>
```